components/RssFeed.php

```diff
@@ -137,8 +137,12 @@ protected function listPosts()
         /*
          * Add a "url" helper attribute for linking to each post and category
          */
-        $posts->each(function($post) {
-            $post->setUrl($this->postPage, $this->controller);
+        $blogPostComponent = $this->getComponent('blogPost', $this->postPage);
+
+        $posts->each(function($post) use ($blogPostComponent) {
```

Comment:
@GinoPane space between function and arguments as per PSR-2 guidelines.
```suggestion
        $posts->each(function ($post) use ($blogPostComponent) {
```